frontend/test/metabase/scenarios/admin/permissions/permissions.cy.spec.js

```diff
@@ -633,3 +654,17 @@ function assertPermissionTable(rows) {
     });
   });
 }
+
+/**
+ * @param {string} permission
+ * @param {boolean} isDisabled
```

Comment:
Total nitpick, doc should probably state that function returns a boolean?

Reply:
I'm never sure how to document the return value when it actually returns the Cypress command (which will then resolve to a certain value, dom element, etc) :/